pkg/Library/Polymath/Atoms/WikipediaContent.js

```diff
@@ -4,16 +4,20 @@ export const atom = (log, resolve) => ({
    * Copyright 2023 NeonFlan LLC
    * SPDX-License-Identifier: BSD-3-Clause
    */
-  async shouldUpdate({query}, state) {
-    return query;
-  },
   async update({query}, state, {service, isDirty}) {
     if (!isDirty('query')) {
       log.warn('query came again!')
       return;
     }
-    const {title, html} = await service('PolymathService', 'QueryWikipedia', {query});
-    return {title, markup: html};
+    if (query) {
+      const {title, html} = await service('PolymathService', 'QueryWikipedia', {query});
+      return {title, markup: html};
+    } else {
+      return {
+        title: undefined,
```

Comment:
As a rule, we never set anything to `undefined`, only to `null`

Reply:
I knew that, i don't know why i used `undefined`, changed to `null` now, thanks